buildbot/master/files/config/master.cfg

```diff
@@ -231,8 +231,22 @@ c['status'] = [
             'failure': 1,
         },
     ),
-]
+    MailNotifier(
+        fromaddr="buildbot@example.org",
+        sendToInterestedUsers=False,
+        extraRecipients=["listaddr@example.org"],
+        builders=[
```

Comment:
I know in the issue I mentioned it would be nice to easily disable builders, but now that Android is back up and running I'm more concerned about this list getting out of sync with the actual nightly builders list. Can you deduplicate these, either by pulling out the list itself or accessing some property on `c['schedulers']`?